### ISSUE TITLE
Changing climate device temperature without changing the mode

### ISSUE THREAD

**Issue by WallK** [nielsfaber/scheduler-component#395]
Is it possible to change temperature of the climate device without changing it's current mode?

### Checklist
- [x] I checked for similar existing requests (both open and closed) before posting.
- [x] My request is generic, other users may benefit from it too.

### Proposal
As a user, I want to change the set temperature of my AC without changing it's state (mode of operation), so I can keep it on, off or in different mode using other automations

### Additional info
When I create a schedule or a scheme -- I can't select temperature without choosing the mode of operation
This forces AC to turn on on the temperature change, for example
Calling **climate.set_temperature** with only **temperature** data does what I expect

**Comment by nielsfaber** [comment]
You can use scheduler to trigger **any** action.
I think you are referring to the built-in action ‘heat’ which both sets the mode to heat and provides a setpoint.
You can extend the actions which are available in the card using the yaml configuration, see https://github.com/nielsfaber/scheduler-card?tab=readme-ov-file#customize for instructions.

**Comment by WallK** [comment]
Yes, I was talking about default set of available actions, there is no "Set temperature" there
![image](https://github.com/user-attachments/assets/4046c5d0-d950-41cf-b8e1-ec39f5328a0c)

Thank you for pointing out the config options, I will play with it

I have investigated other devices in my house and found out that my bathroom floor thermostat will turn on if you change the temperature on it without specifying the mode of operation
So maybe having "Change temperature" built-in action is a bad idea

**Comment by WallK** [comment]
I understand! The current implementation is very obvious as in it does what it says and that's the best solution for most users
Thanks!
